src/test/ts/commands/StatusTest.ts

```diff
@@ -0,0 +1,110 @@
+import * as path from 'path';
+import { describe, it } from 'mocha';
+import { assert } from 'chai';
+import * as O from 'fp-ts/Option';
+import * as Git from '../../../main/ts/utils/Git';
+import * as Files from '../../../main/ts/utils/Files';
+import * as Status from '../../../main/ts/commands/Status';
+import * as BeehiveArgs from '../../../main/ts/args/BeehiveArgs';
+
+const newGit = async (): Promise<Git.TempGit> => {
+  const hub = await Git.initInTempFolder(true);
+  return Git.cloneInTempFolder(hub.dir, O.none);
+};
+
+const branchWithPj = async ({ dir, git }: Git.TempGit, versionString: string, branchName: string): Promise<void> => {
+  await Git.checkoutNewBranch(git, branchName);
+
+  const pjFile = path.join(dir, 'package.json');
+  await Files.writeFile(pjFile, `
+    {
+      "name": "@beehive-test/lifecycle-test",
+      "version": "${versionString}"
+    }
+    `);
```

Comment:
Minor indentation issue here by the looks (should be indented by one more level).

Reply:
Hmmm. It's a multi-line string, so it's always going to be a bit weird. I've outdented the close backtick and paren - I think that looks a bit better.